### PR TITLE
Clean up README formatting.

### PR DESCRIPTION
**Pull request by HugoNikanor**
I changed the typesetting of the numbered lists from code to actual numbered lists.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +35,17 @@ When they receive a bunch of nonsense which does something meaningful, they will
...
-	This will compile your file and create a .bf file with Brainfuck code
-	(optional parameters: -o for output file, and -r for running the compiled file)
-4. Run "Interpreter.py <path_to_bf_file>"
-	This will execute the Brainfuck code
-```
+    * This will compile your file and create a .bf file with
+      Brainfuck code
+    * (optional parameters: `-o` for output file, and `-r`
+      for running the compiled file)
+4. Run "Interpreter.py <path_to_bf_file>`, this will execute
```
[the remark this answers is not in the document]
Comment (HugoNikanor) [reply README.md]: Shouldn't the start `"` instead by changed to ``` ` ```

[the remark this answers is not in the document]
Comment (elikaski) [reply README.md]: I think " looks nicer :)

**Comment by HugoNikanor** [comment]
Now it's a quotation mark instead.

-- 
Hugo Hörnquist

**Comment by elikaski** [comment]
thank you very much 👍

**Pull request merged**